app/src/main/java/com/github/se/travelpouch/ui/home/TravelList.kt

```diff
@@ -186,8 +186,6 @@ fun TravelListScreen(
               floatingActionButton = {
                 FloatingActionButton(
                     onClick = { navigationActions.navigateTo(Screen.ADD_TRAVEL) },
-                    containerColor = MaterialTheme.colorScheme.tertiaryContainer,
-                    contentColor = MaterialTheme.colorScheme.onTertiaryContainer,
```

Comment:
This is not linked to the feature. Please avoid these kind of modifications in your futures features

Reply:
Yes, I am sorry about that. It was not supposed to happen. I thought I corrected it in a previous PR, but I don't know how it slipped